ethcore/wasm/src/env.rs

```diff
@@ -69,17 +69,17 @@ pub const SIGNATURES: &'static [UserFunctionDescriptor] = &[
 	),
 	Static(
 		"_ccall",
-		&[I32; 6],
+		&[I64, I32, I32, I32, I32, I32],
```

Comment:
Is it really a correct signature? In `pwasm-std/pwasm-ethereum` it is defined like

```rust
		pub fn ccall(
			gas: i64,
			address: *const u8,
			val_ptr: *const u8,
			input_ptr: *const u8,
			input_len: u32,
			result_ptr: *mut u8,
			result_len: u32,
		) → i32;
```

which have 7 arguments, but in code above it is only 6.

Reply:
Nice catch, regular call lacks dedicated test also